WordPress/src/main/java/org/wordpress/android/util/AnalyticsUtils.java

```diff
@@ -65,7 +65,7 @@ public static void refreshMetadata(AccountStore accountStore, SiteStore siteStor
      * counts Jetpack sites connected via WPCom Rest API. If there are Jetpack sites in the site store and they're
      * all accessed via XMLRPC, this method returns false.
      */
-    private static boolean isJetpackUser(SiteStore siteStore) {
+    public static boolean isJetpackUser(SiteStore siteStore) {
```

Comment:
Fine for me, I don't think we'll use this in another context, updated in  69de171dc735fd882fa527a42b027d0140f90b09